src/Microsoft.Health.Fhir.Core/Features/KnownHeaders.cs

```diff
@@ -13,6 +13,7 @@ internal static class KnownHeaders
         public const string RetryAfter = "Retry-After";
         public const string ProvenanceHeader = "X-Provenance";
         public const string Prefer = "Prefer";
+        public const string ItemsDeleted = "Items-Deleted";
```

Comment:
>Items-Deleted [](http://example.com/codeflow?start=44&length=13)

since it specific for our server maybe `x-ms-items-deleted`?

Reply:
I could do it for consistency, but I was conflicted by this: https://datatracker.ietf.org/doc/html/rfc6648

Reply:
TIL.

I think it's better to leave it as is.

Maybe it's worth to add comments and break this headers into grops like:

```
  //  FHIR Defined headers
  public const string IfNoneExist = "If-None-Exist";
  public const string ProvenanceHeader = "X-Provenance";
 // Generic headers
        public const string RetryAfter = "Retry-After";
        public const string Prefer = "Prefer";
        public const string RequestId = "X-Request-Id";
        public const string CorrelationId = "X-Correlation-Id";
// Microsoft defined headers
     public const string RetryAfterMilliseconds = "x-ms-retry-after-ms";
// Headers our service uses
// Please try to not use x-ms convention because it's obsolete: [https://datatracker.ietf.org/doc/html/rfc6648](https://datatracker.ietf.org/doc/html/rfc6648)
        public const string PartiallyIndexedParamsHeaderName = "x-ms-use-partial-indices";
          public const string ItemsDeleted = "Items-Deleted";
        public const string EnableChainSearch = "x-ms-enable-chained-search";
        public const string ProfileValidation = "x-ms-profile-validation";

```